### PR TITLE
XS-4561 Update version and added disclosure check

### PR DESCRIPTION
**Pull request by allenbross-wf**
### Changes:

- Changed version
- Added check to verify EFM disclosure is used.

This needs to go with https://github.com/DataQualityCommittee/dqc_us_rules/pull/241, https://github.com/DataQualityCommittee/dqc_us_rules/pull/238, and https://github.com/DataQualityCommittee/dqc_us_rules/pull/239

### Pull request process

- [ ] Full description of changes provided above.
- [ ] Reviewed by another person. (+1)
- [ ] If this is a code change, reviewed by a second person. (+1)
- [ ] QA'd to the specifications listed [here](https://github.com/DataQualityCommittee/dqc_us_rules#quality-assurance-qa-of-a-pull-request). Documentation only changes do not require the formal code quality assurance process. (+10)

### CODE REVIEW

**Comment by mindeerickson-wf** [comment]
+1

File: dqc_us_rules/__init__.py [hunk 1]
```diff
@@ -10,6 +10,14 @@ def run_checks(val, *args, **kwargs):
     """
     The function to run all of the validation checks under the SEC package
     """
+    if val.disclosureSystem.validationType != "EFM":
```
Comment (andrewperkins-wf) [inline dqc_us_rules/__init__.py]: Shouldn't this be something like 
```python
if 'EFM' in val.disclosureSystem.validationType
```

To handle things like efm-strict? Or is that handled by the disclosure system behavior already?

**Comment by andrewperkins-wf** [comment]
pass complete

File: dqc_us_rules/__init__.py (hunk 1 above)
Comment (allenbross-wf) [inline dqc_us_rules/__init__.py]: As coded the check works because efm-strict is of validationType EFM.

Comment (andrewperkins-wf) [inline dqc_us_rules/__init__.py]: got it - thanks

**Comment by andrewperkins-wf** [comment]
+1

**Comment by andrewperkins-wf** [comment]
QA +10
* Verified that the new error fires when using another disclosure system (used EBA's EIOPA disclosure system)
* Verified that it does not fire when using an EFM disclosure system (used efm-strict)

**Comment by andrewperkins-wf** [comment]
All reviews completed, QA finished. Merging.

**Pull request merged**